pvm.py

```diff
@@ -30,18 +31,40 @@ def _init_logger():
 VIDEO_PATH = "jellyfish720p.mp4"
 media = ""
 IS_FILE_SET = False
+NEXT_TIME = None
 
 def parse_commands(*args):
 	global media
 	global VIDEO_PATH
 	global IS_FILE_SET
+	global NEXT_TIME
+	if len(args) == 4:
+		seconds = int(args[3]) + 3
+		mintues = int(args[2])
+		hours = int(args[1])
+		if seconds >= 60:
+			seconds -= 60
+			mintues += 1
+		if mintues == 60:
+			mintues = 0
+			hours += 1
+		time_data = str(hours) + ":" + str(mintues) + ":" + str(seconds)
+		NEXT_TIME = datetime.strptime(time_data, "%H:%M:%S")
+		_logger.info("Next action will be done in %s", time_data)
+		return
 	command = args[1]
 	_logger.info("Received command: %s", command)
 	if len(args)>2:
 		value = args[2]
-		_logger.info("Received command: %s", str(value))
+		_logger.info("Received value: %s", str(value))
 		pass
 	# TODO: Create another python file to control two display
+	while True:
+		now = datetime.now()
+		now = now.replace(microsecond=0)
+		if now.time() == NEXT_TIME.time():
+			break
+		sleep(0.01)
```

Comment:
interesting... does that mean that the rpi will become unresponsive for 3 seconds?
also, it seems that if for some reason the clock of the rpi device is not in sync with the control computer then chaos will be easily instaured: e.g. if the timestamp is for 18:00:00 (to which this script will add 3 seconds) and the rpi is at 18:00:04, it means that the target time will never reach and the rpi device will become permanently unresponsive until it is manually reset?

Reply:
I haven't tested the situation in RPIs where press start and pause in the 3s.
But from the logic of the code, it will be permanently unresponsive. Fix this in the newest commit.